Fix for passing test

YukJiSoo opened the pull request:
## Summary 

## 리뷰 참고 사항

- workflow/testing.yml 에 등록된 절차를 통과하지 못하는 이슈 해결

### lint:js
- https://github.com/line/webpack.kr/pull/10/commits/b9020facbe28e99468f3624505ac7b5879681bb8
- 에러 내용
![image](https://user-images.githubusercontent.com/30258523/121469846-2c76fd00-c9f8-11eb-89b4-84f8046d4dd7.png)
- Edit button을 원복하는 것으로 해결

### lint:markdown
- https://github.com/line/webpack.kr/pull/10/commits/6e174fa642c9bccd83a81a5f34c981a4bb2c0532
- 커맨드 실행 시 lint 에러가 발생하던 부분 syntax 수정

### cypress
- https://github.com/line/webpack.kr/pull/10/commits/8aa6b491992fa4a14926f3e42cb64b819bfde679
- 에러 내용
![image](https://user-images.githubusercontent.com/30258523/121471264-516c6f80-c9fa-11eb-9854-88a02c1a59d9.png)
- meta tag에 들어가는 내용을 번역한 부분도 cypress에서 체크해주고 있기 때문에 번역된 내용으로 수정

### lint:links
- 에러 내용
![image](https://user-images.githubusercontent.com/30258523/121470350-0bfb7280-c9f9-11eb-94b2-35906de0fd16.png)
- `lint:links`: 빌드된 html 파일에서 잘못된 형식의 링크가 있는지 검사하는 커맨드
- https://github.com/line/webpack.kr/pull/10/commits/f8e2d8b1928e3bd26d96ae42bada1854a1d03793
  -  잘못된 이름으로 추가된 fragment 수정
  - as-is: optimizationoccurrenceorder
  - to-be: optimizationprovidedexports
- https://github.com/line/webpack.kr/pull/10/commits/abafa0a464a5be96703fe87573dd0cd12ab573d3
  - `/#sponsor` 로 이동해야 하지만 번역 시 메인 페이지에서 sponsor 부분을 제거
  - webpack 영문사이트로 이동하도록 수정
- https://github.com/line/webpack.kr/pull/10/commits/a2e3c2a11824d9c986aaa7e516fa33575f7912df
  - 문제
    - index.md에서 ##을 한글로 번역한 부분이 빌드 과정에서 html entity로 encoding 되어서 a tag의 href 속성에 추가됨
    - 변환된 index.html에서 id 값과 url의 fragment 값이 일치하지 않아서 lint:link 커맨드 실행 시 fail case 발생
  - 해결
    - index page에서 ##을 사용하지 않고 제목으로 보이게 스타일만 추가하는 것으로 해결

YukJiSoo committed:
Restore edit button

YukJiSoo committed:
Change md file content's syntax for passing test

YukJiSoo committed:
Change cypress test set's expected value for passing test

YukJiSoo committed:
Fix uncorrect url fragment

YukJiSoo committed:
Change to original docs site's sponsor link

- 한국어 번역 사이트에서는 sponsor를 노출하고 있지 않아 테스트에 실패하기 때문에 원본 페이지로 이동하도록 수정

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/line/webpack.kr?pullRequest=10) <br/>All committers have signed the CLA.

dkstyle reviewed (approved):
👍

YukJiSoo committed:
Replace h tag to div for skipping lint:link command

문제
- index.md에서 ##을 한글로 번역한 부분이 빌드 과정에서 html entity로 encoding 되어서 a tag의 href 속성에 추가됨
- 변환된 index.html에서 id 값과 url의 fragment 값이 일치하지 않아서 lint:link 커맨드 실행 시 fail case 발생

해결
- 근본적인 해결을 위해서는 사용되는 remark-slug 라이브러리 수정이 필요함
- index page에서 ##을 사용하지 않고 제목으로 보이게 스타일만 추가

YukJiSoo committed:
Remove redundant NpmInstallWebpackPlugin content in plugin page

YukJiSoo commented:
https://github.com/line/webpack.kr/pull/10/commits/29bcad1fcd9455f3b72f71271bce44b262d2fb3c
- 에러 내용
![image](https://user-images.githubusercontent.com/30258523/121642017-e6d73480-caca-11eb-93eb-3c9d3c949339.png)
- 존재하지 않는 문서에 대한 링크가 추가되어 있어서 plugin index page에서 해당 부분 제거

chenxsan commented:
> [29bcad1](https://github.com/line/webpack.kr/commit/29bcad1fcd9455f3b72f71271bce44b262d2fb3c)
> 
> * 에러 내용
>   ![image](https://user-images.githubusercontent.com/30258523/121642017-e6d73480-caca-11eb-93eb-3c9d3c949339.png)
> * 존재하지 않는 문서에 대한 링크가 추가되어 있어서 plugin index page에서 해당 부분 제거

It was renamed to `install-webpack-plugin`. We already fixed it upstream https://github.com/webpack/webpack.js.org/pull/5069.

YukJiSoo committed:
Rename NpmInstallWebpackPlugin's url

https://github.com/line/webpack.kr/pull/10#issuecomment-859996892

keipark reviewed (approved):
👍 👍

YukJiSoo committed:
Change Translators section's className

- cypress에서 contributors__section className을 사용하고 있음
- contributors__section 이 한 페이지에 중복으로 사용되는 경우 test가 실패하기 때문에 Translators 인경우 translators__section을 사용

YukJiSoo commented:
![image](https://user-images.githubusercontent.com/30258523/122498717-94e26180-d02a-11eb-8607-5ecad447c4f2.png)
https://github.com/line/webpack.kr/pull/10/commits/927a568eeeb087a0cc0038b2316dc46e3a9b6e6f
- 새로 추가된 Translator로 인해 cypress 테스트가 실패하는 것 수정
